performance-tests/perf-tests.sh

```diff
@@ -29,17 +33,28 @@ KAFKA_TOOL_IMAGE=${KAFKA_TOOL_IMAGE:-quay.io/strimzi/kafka:${STRIMZI_VERSION}-ka
 KAFKA_IMAGE=${KAFKA_IMAGE:-"quay.io/ogunalp/kafka-native:latest-kafka-${KAFKA_VERSION}"}
 KROXYLICIOUS_IMAGE=${KROXYLICIOUS_IMAGE:-"quay.io/kroxylicious/kroxylicious:${KROXYLICIOUS_VERSION}"}
 PERF_NETWORK=performance-tests_perf_network
-export KAFKA_VERSION KAFKA_TOOL_IMAGE KAFKA_IMAGE KROXYLICIOUS_IMAGE
+CONTAINER_ENGINE=${CONTAINER_ENGINE:-"docker"}
+export KAFKA_VERSION KAFKA_TOOL_IMAGE KAFKA_IMAGE KROXYLICIOUS_IMAGE CONTAINER_ENGINE
+
+
+
+printf "KAFKA_VERSION: ${KAFKA_VERSION}\n"
+printf "STRIMZI_VERSION: ${STRIMZI_VERSION}\n"
+printf "KROXYLICIOUS_VERSION: ${KROXYLICIOUS_VERSION}\n"
+printf "KAFKA_IMAGE: ${KAFKA_IMAGE}\n"
+printf "KROXYLICIOUS_IMAGE: ${KROXYLICIOUS_IMAGE}\n"
+
 
 runDockerCompose () {
```

Comment:
nit: we should maybe rename this to `runCompose`

Reply:
Everybody calls it "docker compose" anyway and it doesn't work on podman due to --wait, so I think it's safe to keep it as it is. If I read `runCompose` it wouldn't mean much to be (albeit semantically correct :))